### PR TITLE
Make custom props list usable

### PR DESCRIPTION
**Pull request by soljian**
I had to make your package working with my custom props list so here are my corrections. 

It's fully working for me. (I didnt added the images but it's ok)

image

### CODE REVIEW

**Comment by AliLogic** [comment]
Thank you so much for the amazing pull request.

I will look through the pull requests, and request any changes if necessary before this is merged to the main codebase.  Thanks again.

File: client/editor.lua [hunk 1]
```diff
@@ -121,9 +121,10 @@ function Editor_HandleCreateObject(x, y, z)
 end
 
 local function Editor_CreateObjectPlacement(objectID, rx, ry, rz, sx, sy, sz)
+	print(objectID, rx, ry, rz, sx, sy, sz)
```
Comment (AliLogic) [inline client/editor.lua]: remove this?

**Review by AliLogic — CHANGES_REQUESTED** [review]

File: client/editor.lua (hunk 1 above)
Comment (soljian) [reply client/editor.lua]: Oups, done

**Review by AliLogic — APPROVED** [review]

**Comment by AliLogic** [comment]
Thanks!

**Pull request merged**